models/articles.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+exports.selectArticlesById = (articleID) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1;", [articleID])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: `No user found by this ID`,
```

Comment:
It is not a user ID, but an article ID. Error message needs fixing to be more appropriate.